src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,153 @@
+import { Avatar, Box, Button } from '@mui/material';
+
+import { Place, Facebook, Layers, StarRate, People, Person } from '@mui/icons-material';
+
+import { styled } from '@mui/material/styles';
+import Badge from '@mui/material/Badge';
+
+// import { useSelector } from 'react-redux';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import localAvatar from '../assets/images/avatar.png';
+import useStyles from './styles';
+
+import { fetchProfileAction } from '../../features/profile/profileSlice';
+
+import { useAppDispatch } from '../../app/store';
+import { IStoreDataTypes } from '../../app/types';
+
+const StyledBadge = styled(Badge)(({ theme }) => ({
+  '& .MuiBadge-badge': {
+    backgroundColor: '#44b700',
+    color: '#44b700',
+    boxShadow: `0 0 0 2px ${theme.palette.background.paper}`,
+    '&::after': {
+      position: 'absolute',
+      top: 0,
+      left: 0,
+      width: '100%',
+      height: '100%',
+      borderRadius: '50%',
+      animation: 'ripple 1.2s infinite ease-in-out',
+      border: '1px solid currentColor',
+      content: '""',
+    },
+  },
+  '@keyframes ripple': {
+    '0%': {
+      transform: 'scale(.8)',
+      opacity: 1,
+    },
+    '100%': {
+      transform: 'scale(4.4)',
+      opacity: 0,
+    },
+  },
+}));
+
+function Profile() {
+  const classes = useStyles();
+
+  const padding = 20;
+  const [user, setUser] = useState('Chayuga');
+
+  // dispatch
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(fetchProfileAction(user));
+  }, [dispatch]);
+
+  const store = useSelector((state:IStoreDataTypes) => state?.profile);
+
+  const { loading, profile } = store;
+
+  return (
+    <div>
+      {loading ? <h1 className="text-green-300 text-3xl text-center">Loading please wait...</h1>
+        : (
+          <div>
+            <div className={classes.profile}>
+              <Box display="flex" flexDirection="column" justifyContent="space-around" alignItems="center">
+
+                <StyledBadge
+                  overlap="circular"
+                  anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
+                  variant="dot"
+                >
+                  <Avatar
+                    alt="Profile"
+                    src={profile?.avatar_url ? profile?.avatar_url : localAvatar}
+                    sx={{ width: 165, height: 165 }}
+
+                  />
+                </StyledBadge>
+
+                <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'center', textAlign: 'center', paddingTop: padding }}>
```

Comment:
Why are we moving to inline styles all of a sudden? This also seems to be the only component to do so. Copy paste?

Reply:
No, not copy-paste, I was working to get the design up and everything seemed to not work, so am transferring the styles to a separate file.

Reply:
Ah okay awesome. I was so confused as it deviated so much from what you've been doing up until this point.